src/libcollections/linked_list.rs

```diff
@@ -51,8 +51,8 @@ struct Rawlink<T> {
 }
 
 impl<T> Copy for Rawlink<T> {}
-unsafe impl<T:'static+Send> Send for Rawlink<T> {}
-unsafe impl<T:Send+Sync> Sync for Rawlink<T> {}
+unsafe impl<T: Send> Send for Rawlink<T> { }
+unsafe impl<T: Sync> Sync for Rawlink<T> { }
```

Comment:
As a passing idea, these may also be best done through `Unique` instead of an explicit `unsafe impl`. I'm not quite sure if the unique semantics though are correct, I'm not super familiar with this implementation.